client/state/happychat/selectors/test/get-happychat-timeline.js

```diff
@@ -16,23 +16,17 @@ describe( '#getHappychatTimeline', () => {
 	const ONE_MINUTE = 1000 * 60;
 	const FIVE_MINUTES = ONE_MINUTE * 5;
 	const timelineAtTime1 = [
-		{ timestamp: ( NOW - FIVE_MINUTES ) / 1000, id: '1-1' },
-		{ timestamp: ( NOW - ONE_MINUTE ) / 1000, id: '1-2' },
-		{ timestamp: NOW / 1000, id: '1-3' },
+		{ timestamp: NOW - FIVE_MINUTES, id: '1-1' },
```

Comment:
Just curious: what does this mean? I mean in general the `timelineAtTime`.

Reply:
As I understand it, the chat client loads a "timeline" of chat events to display to the customer from the server side and caches it. This test suite simulates various odd situations such as two differing timeline query results and tries to ensure the selector behaves as desired.

Reply:
Oh, interesting. Thanks!